src/lib/ngsi/AttributeDomainName.cpp

```diff
@@ -99,11 +99,11 @@ void AttributeDomainName::present(const std::string& indent)
 {
   if (string != "")
   {
-    PRINTF("%sAttributeDomainName: %s\n", indent.c_str(), string.c_str());
+    LM_F(("%sAttributeDomainName: %s", indent.c_str(), string.c_str()));
```

Comment:
Yes, all of them must and will be replaced.
I'll create an issue about it.

I used these 'present()' methods for debugging when the parser was implemented and with the broker running in foreground. It was easier that way.
However, things have changed and now all testing is done via functests and it is now necessary to see the result of 'present' is the log file.